settings.gradle

```diff
@@ -26,7 +26,8 @@ plugins {
 }
 
 rootProject.name = "opentelemetry-java"
-include ":api:all",
+include ":all",
```

Comment:
What is the purpose of the `all` project anyway?

Reply:
Oh, I see it has this archunit test and the coverage report collection... Maybe adding a short README to the subfolder would be helpful? I think it's not that obvious.

Reply:
Sure